user_manual/changes/U816-006.yaml

```diff
@@ -0,0 +1,9 @@
+type: api-change
+title: Fix parser for pragma argument assocs
```

Comment:
I think it should be clearer that this breaks `PragmaArgumentAssoc` in the title, since this is an entry for an api-change, we care less about the fix than about the API change.